src/main/java/com/app/onlinebookstore/security/JwtUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.app.onlinebookstore.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Date;
+import java.util.function.Function;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtUtil {
+    private final Key secret;
+    @Value("${jwt.expiration}")
+    private long expiration;
+
+    public JwtUtil(@Value("${jwt.secret}") String secretString) {
+        secret = Keys.hmacShaKeyFor(secretString.getBytes(StandardCharsets.UTF_8));
+    }
+
+    public String generateToken(String username) {
+        return Jwts.builder()
+                .setSubject(username)
+                .setIssuedAt(new Date(System.currentTimeMillis()))
```

Comment:
new Date() == new Date(System.currentTimeMills())
```suggestion
                .setIssuedAt(new Date())
```

Reply:
I found that they have some diferents